Makefile

```diff
@@ -82,6 +82,11 @@ install:
 	install -D -m0755 buildah $(DESTDIR)/$(BINDIR)/buildah
 	$(MAKE) -C docs install
 
+.PHONY: uninstall
+uninstall:
+	rm -f $(DESTDIR)/$(BINDIR)/buildah
+	rm -f $(PREFIX)/share/man/man1/buildah*.1
```

Comment:
What about completions and other stuff that you installed?

Reply:
We only install these two with `make install`.  Should this remove everything that we could potentially install other commands like `make install.completions`?  @rhatdan should we add completions to the standard `make install`?

Reply:
No I don't think so, but the reason for a make uninstall is to remove cruft from your system, I guess although it is seldom done.
If we are going to add a make uninstall, we should have either a make uninstall.completions... or just have make uninstall all of the stuff that the Makefile could have added.

Reply:
updated, Not sure if it is correct.